pkg/operator/ceph/object/rgw_test.go

```diff
@@ -122,6 +122,9 @@ func simpleStore() *cephv1.CephObjectStore {
 			MetadataPool: cephv1.PoolSpec{Replicated: cephv1.ReplicatedSpec{Size: 1, RequireSafeReplicaSize: false}},
 			DataPool:     cephv1.PoolSpec{ErasureCoded: cephv1.ErasureCodedSpec{CodingChunks: 1, DataChunks: 2}},
 			Gateway:      cephv1.GatewaySpec{Port: 123},
+			Network: cephv1.NetworkSpec{
+				HostNetwork: true,
```

Comment:
This is the preferred setting for host networking. The `hostNetwork: true` is legacy.
```suggestion
				Provider: "host",
```

Reply:
no needed to modify this, it's only suit for testcase.